jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -588,7 +589,7 @@ static JavaContainerBuilder getJavaContainerBuilderWithBaseImage(
                 "mainClass, extraClasspath, jvmFlags, and expandClasspathDependencies "
                     + "are ignored for WAR projects"));
       }
-      return null;
+      return Arrays.asList("java", "-jar", "/usr/local/jetty/start.jar");
```

Comment:
Let's update the Javadoc to mention this

Reply:
And just to make sure I'm understanding this correctly, what does this new entrypoint do exactly?

Reply:
It's the command to start the Jetty server. Jetty will start running and serving a deployed WAR app.

Reply:
Cool! Just out of curiosity - why was this entrypoint null before?

Reply:
Oh it was inherited from the base image